src/web/components/elements/YoutubeBlockComponent.tsx

```diff
@@ -1,10 +1,33 @@
 import React from 'react';
+import { css } from 'emotion';
+
+import { palette } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
 
 import { Caption } from '@root/src/web/components/Caption';
 import { Display } from '@root/src/lib/display';
 import { YoutubeAtom } from '@guardian/atoms-rendering';
 import { asFormat } from '@root/src/lib/format';
 
+const expiredFallbackTextStyles = css`
+    position: absolute;
+    display: table-cell;
+    padding: 1.125rem 3.75rem;
+    ${textSans.medium({ fontWeight: 'bold', lineHeight: 'tight' })}
```

Comment:
Think that's the wrong font-face 
![image](https://user-images.githubusercontent.com/638051/93580999-7286bf00-f998-11ea-9624-5f7a2db57d1d.png)

Reply:
I think probably worth just making this consistent and use an icon from the Design System? https://theguardian.design/2a1e5182b/p/3892a2-icons

Not sure if alert or info is better though @SiAdcock @HarryFischer ?

Reply:
I was hoping there was a way of use `textSans` to set the font of the text, But I was unable to find how to do it. Should I instead custom set it in the CSS? or is there a lib/pattern I could follow to set it?

Reply:
I think you just want to use `body` from the design system https://theguardian.design/2a1e5182b/p/95d5d0-code/t/454e6d